client/stylesheets/_toCat.scss

```diff
@@ -5153,6 +5153,11 @@ On click the images will be displayed at normal size to complete the effect
 
 /* Settings */
 
+.profileSettingsUl {
+    width: 70% !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered margin, width